packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js

```diff
@@ -58,4 +61,71 @@ export default class PciStoragesContainersUsersController {
       type: 'navigation',
     });
   }
+
+  static isActionsAvailable(container) {
+    const Ctrl = PciStoragesContainersUsersController;
```

Comment:
**nitpick:** Can we rename the controller class name to `PciStoragesContainersController`? the file indicates just containers and not related to users.

Reply:
completed